client/src/Components/CarList.js

```diff
@@ -1,31 +1,41 @@
-import React from "react";
+import React, { useState } from "react";
 import { useQuery } from "@apollo/react-hooks";
-
+import Select from 'react-select';
 import { getUserDataQuery } from "../queries/queries";
 import LoadingSpinner from "./LoadingSpinner";
 
 function displayCars(data) {
-  return data.cars.map(car => {
-    return (<li key={car.id}>
-      {car.displayName}({car.make} - {car.model})
-    </li>);
+   return data.cars.map(car => {
+      return { value: `${car.id}`, label: `${car.displayName} ${car.make} (${car.model})` }
   });
 }
 
 export default function() {
+  const [selectedOption , setSelectedOption] = useState(null)
```

Comment:
need to lift the state up to the wrapper component. Also few little code style issues. Can you install the vs code prettier extension? We should probably add eslint to this codebase